.github/workflows/move_crd_values_to_operator.yaml

```diff
@@ -0,0 +1,109 @@
+name: Move Operator CRD Manifest File To Operator
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  check-specific-file:
+    runs-on: ubuntu-latest
+    outputs:
+      CHANGED: ${{ steps.check_changes.outputs.CHANGED }}
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Get Source Branch Name
+        id: get_source_branch
+        run: |
+          MERGE_COMMIT=$(git --no-pager log -1 --merges --pretty=%H)
+          SOURCE_BRANCH=$(git show -s --pretty=%B $MERGE_COMMIT | grep -oP '(?<=from )\S+' | sed 's/^[^/]*\///')
+          echo "SOURCE_BRANCH=${SOURCE_BRANCH}" >> $GITHUB_ENV
+
+          if [[ "$SOURCE_BRANCH" == *operator-release* ]]; then
+            echo "Branch is a release 'release', exiting pipeline."
+            exit 0
+          fi            
+
+      - name: Check for Changes in components/tyk-operator/templates/all.yaml
+        id: check_changes
+        run: |
+          echo "Checking for changes between in ${{ env.SOURCE_BRANCH }} branch"
+          git fetch origin ${{ env.SOURCE_BRANCH }}
+          if git --no-pager log -n 1 origin/${{ env.SOURCE_BRANCH }} --name-only --pretty=format: | grep -q "components/tyk-operator/templates/all.yaml"; then
```

Comment:
As we are syncing changes in values.yaml and Chart.yaml too, should this condition be changed?